fixperms.sh

```diff
@@ -66,6 +66,10 @@ fixperms () {
     tput sgr0
     
     #Fix individual files in public_html
+
+
+    chown -R $verbose $account:$account $HOMEDIR/public_html
```

Comment:
There is already a `chown` for the 'public_html' directory in the script. https://github.com/PeachFlame/cPanel-fixperms/blob/master/fixperms.sh#L82

```bash
chown $verbose $account:nobody $HOMEDIR/public_html
```

This `chown` you are introducing would get overridden. Is the correct ownership supposed to be  USER:USER or USER:nobody ?

I no longer have access to test a cPanel server anymore. I run pure vanilla Linux without cPanel these days. So I would need to rely more on the community to ensure this change is right.

Reply:
public_html directory itself should be $account:nobody but the content in it will be $account:$account

I have tested this in a 500 user cPanel server and works just fine.

The problem was the content in public_html does not get owner and apache/litespeed will return 403 because its running suphp and does not have privilege to run the code.

Example error: owner of file does not match owner of vhost, path